src/communication/kernels/broadcast.cpp

```diff
@@ -50,8 +50,13 @@ template <class CommSender, class TileSender>
   // The input tile must be copied to the temporary tile used for the send, but
   // the temporary tile does not need to be copied back to the input since the
   // data is not changed by the send. A send does not require contiguous memory.
+  constexpr bool require_contiguous =
```

Comment:
Do you mean
```suggestion
  constexpr auto require_contiguous =
```
?

Reply:
Yep... good catch.

Reply:
https://github.com/eth-cscs/DLA-Future/pull/1096/commits/0c0fea8fb7db11ec65a2b154aa8f8f8b920931fd